recipes/clickclick/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install 
```

Comment:
Please remove `bld.bat` and `build.sh` files in favor of using the `script` specification in the meta.yaml:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L26
https://conda.io/docs/building/meta-yaml.html#script